plugin/pkg/auth/authorizer/rbac/rbac_test.go

```diff
@@ -44,18 +46,18 @@ func newRole(name, namespace string, rules ...rbacv1.PolicyRule) *rbacv1.Role {
 	return &rbacv1.Role{ObjectMeta: metav1.ObjectMeta{Namespace: namespace, Name: name}, Rules: rules}
```

Comment:
Role is namespace-scoped, so it "Tenant" should be set.

Reply:
fixed